app/src/main/AndroidManifest.xml

```diff
@@ -719,7 +719,7 @@
             android:label="@string/pdf_viewer_title"
             android:parentActivityName=".component.ui.plan.PlansActivity" />
         <activity
-            android:name=".component.ui.tufilm.show_ticket"
+            android:name=".component.ui.ticket.ShowTicketActivity"
             android:configChanges="orientation|screenSize"
             android:label="@string/your_ticket"
             android:launchMode="singleTop"
```

Comment:
I think we should set the KinoActivity as ParentActivity. As user, I would expect the back button to bring me to the KinoActivity again and not the MainActivity...
\<activity
            android:name=".component.ui.ticket.ShowTicketActivity"
            android:configChanges="orientation|screenSize"
            android:label="@string/your_ticket"
            android:launchMode="singleTop"
            android:parentActivityName=".component.ui.tufilm.KinoActivity">
            <meta-data
            android:name="android.support.PARENT_ACTIVITY"
            android:value=".component.ui.tufilm.KinoActivity"/>
        </activity>